### PR TITLE
Sample rhdevdocs 3866

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
[RHDEVDOCS-3866](https://issues.redhat.com/browse/RHDEVDOCS-3866): Doc updates-Update Developer Perspective content based on recognized user/Merging the accessing perspectives sections
OpenShift 4.11 Freeze
JIRA issues: [RHDEVDOCS-3866](https://issues.redhat.com/browse/RHDEVDOCS-3866)

SME Review: @jerolimov
QE Review: @sanketpathak
Peer Review: @rh-tokeefe 

Doc Preview: https://54197--docspreview.netlify.app/openshift-enterprise/latest/web_console/web-console-overview.html

Note: Created sample branch because the changes were not getting updated in the old PR.
Old PR Link: https://github.com/openshift/openshift-docs/pull/52924

### CODE REVIEW

File: modules/odc-accessing-developer-perspective.adoc [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-
-.Prerequisites
-To access the *Developer* perspective, ensure that you have logged in to the web console. The default view for the {product-title} web console is the *Developer* perspective.
-
-.Procedure
-
-. Use the perspective switcher to switch to the *Developer* perspective. The *Topology* view with a list of all the projects in your cluster is displayed.
-+
-.Developer perspective
-image::web_console_perspectives.png[The administrator and developer perspectives in the {product-title} web console.]
```
Comment (jerolimov) [inline modules/odc-accessing-developer-perspective.adoc:21]: You don't want show this image anymore?

web_console_perspectives

Comment (shreyasiddhartha) [reply modules/odc-accessing-developer-perspective.adoc:21]: Hi @jerolimov. Yes, we decided to remove this image as other similar sections did not have such images. This was removed just to maintain consistency.

**Review by christoph-jerolimov — APPROVED** [review]
Lgtm!

**Comment by cbremble** [comment]
Per this conversation: https://redhat-internal.slack.com/archives/GR18314R3/p1674584906201279?thread_ts=1673531450.095519&cid=GR18314R3 this should also go to 4.9 and 4.10.

**Comment by rolfedh** [comment]
@shreyasiddhartha: In case you see this first, the two preceding cherry-pick attempts failed because there is a merge issue. To resolve these merge issues, please perform two manual cherry-pick operations, one for each branch, by using the instructions mentioned in the related Slack thread.

**Pull request merged**